README.md

```diff
@@ -104,12 +104,13 @@ After action execution it will return the test counts as output.
 
 A full set list of possible output values for this action.
 
-| **Output**            | **Description**                                                                        |
-|-----------------------|----------------------------------------------------------------------------------------|
-| `outputs.total`       | The total number of test cases covered by this test-step.                              |
-| `outputs.passed`      | The number of passed test cases.                                                       |
-| `outputs.skipped`     | The number of skipped test cases.                                                      |
-| `outputs.failed`      | The number of failed test cases.                                                       |
+| **Output**            | **Description**                                           |
+|-----------------------|-----------------------------------------------------------|
+| `outputs.total`       | The total number of test cases covered by this test-step. |
+| `outputs.passed`      | The number of passed test cases.                          |
+| `outputs.skipped`     | The number of skipped test cases.                         |
+| `outputs.failed`      | The number of failed test cases.                          |
+| `outputs.failedTests` | Names and descriptions of failed tests.                   |
```

Comment:
Nit: We could try to keep the changes minimal an only append a line to make merges smoother.

Reply:
Good point!
https://github.com/bitmovin-engineering/action-junit-report/pull/1/commits/881b5206923bbf3ee46749f7fe044ebbe32290d6